### ISSUE TITLE
I want to test the accuracy of a specific class. How do I know which number represents said class?

### ISSUE THREAD

**Issue by Wazhee** [YueYANG1996/LaBo#26]

**Comment by YueYANG1996** [comment]
You can get the list of class names from the following scripts. The index in this list is the number representing a class.

```
import json
import torch
import argparse
import numpy as np
from models.asso_opt.asso_opt import AssoConcept

parser = argparse.ArgumentParser()
parser.add_argument('--ckpt_path', type=str, required=True)
args = parser.parse_args()

# load ckpt
model = AssoConcept.load_from_checkpoint(str(args.ckpt_path))

cfg = model.cfg
class_names = np.load(cfg.cls_name_path).tolist()
```
